src/client/components/GameCardComponent/ButtonComponent/ButtonComponent.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import './ButtonComponent.styles.css';  
+
+
+
+export default function ButtonComponent({buttonTitle}) {
```

Comment:
Would consider extending the button with more types or renaming the button to `SubmitButton` or `FormSubmitButton`so this button will only be useful in a form as it has `type="submit"`